grammar/SubstraitLexer.g4

```diff
@@ -0,0 +1,157 @@
+lexer grammar SubstraitLexer;
+
+// Whitespace and comment handling
+LineComment   : '//' ~[\r\n]* -> channel(HIDDEN) ;
+BlockComment  : ( '/*' ( ~'*' | '*'+ ~[*/] ) '*'* '*/' ) -> channel(HIDDEN) ;
+Whitespace    : [ \t\r]+ -> channel(HIDDEN) ;
+
+// Substrait is case-insensitive, ANTLR is not. So, in order to define our
+// keywords in a somewhat readable way, we have to define these shortcuts.
+
+fragment A : [aA];
+fragment B : [bB];
+fragment C : [cC];
+fragment D : [dD];
+fragment E : [eE];
+fragment F : [fF];
+fragment G : [gG];
+fragment H : [hH];
+fragment I : [iI];
+fragment J : [jJ];
+fragment K : [kK];
+fragment L : [lL];
+fragment M : [mM];
+fragment N : [nN];
+fragment O : [oO];
+fragment P : [pP];
+fragment Q : [qQ];
+fragment R : [rR];
+fragment S : [sS];
+fragment T : [tT];
+fragment U : [uU];
+fragment V : [vV];
+fragment W : [wW];
+fragment X : [xX];
+fragment Y : [yY];
+fragment Z : [zZ];
+
+fragment DIGIT: [0-9];
+
+fragment INTEGER
```

Comment:
This fragment doesn't appear to be used elsewhere.

Reply:
Removed it. I was using this for TestCase file format.